plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -54,39 +54,39 @@ module.exports = ({containerConfig, appConfig}) => {
       </Row>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {!!fields.resources.cpus && (
```

Comment:
I'd vote for verbose conditions instead of the `!!value` notation to improve readability/maintainability. Let's change this condition to `(fields.resources.cpus != null && fields.resources.cpus >  0)`.